server/services/__tests__/topicsService.spec.js

```diff
@@ -18,70 +17,33 @@ describe('Topics Service', () => {
   });
 
   describe('getTopics', () => {
-    it('returns topics from both sources', async () => {
-      hubMenuRepo.tagsMenu.mockResolvedValue([createTopic('Hub menu topic')]);
-      cmsApi.primaryMenu.mockResolvedValue([createTopic('Primary menu topic')]);
+    it('returns topics', async () => {
+      cmsApi.getTopics.mockResolvedValue([createTopic('topic')]);
 
-      const result = await topicService.getTopics(1);
+      const result = await topicService.getTopics('berwyn');
 
       expect(result).toStrictEqual([
         {
-          description: 'Hub menu topic Desc',
+          description: 'topic Desc',
           href: '/content/1',
           id: '1',
-          linkText: 'Hub menu topic',
-        },
-        {
-          description: 'Primary menu topic Desc',
-          href: '/content/1',
-          id: '1',
-          linkText: 'Primary menu topic',
+          linkText: 'topic',
         },
       ]);
     });
 
-    it('sorts topic by link text ignoring case', async () => {
-      hubMenuRepo.tagsMenu.mockResolvedValue([
-        createTopic('Ad'),
-        createTopic('Aa'),
-        createTopic('ab'),
-      ]);
-      cmsApi.primaryMenu.mockResolvedValue([createTopic('Ac')]);
+    it('Source to have been called correctly', async () => {
+      cmsApi.getTopics.mockResolvedValue([]);
 
-      const result = await topicService.getTopics(1);
+      await topicService.getTopics('berwyn');
 
-      expect(result.map(r => r.linkText)).toStrictEqual([
-        'Aa',
-        'ab',
-        'Ac',
-        'Ad',
-      ]);
-    });
-
-    it('Sources to have been called correctly', async () => {
-      hubMenuRepo.tagsMenu.mockResolvedValue([]);
-      cmsApi.primaryMenu.mockResolvedValue([]);
-
-      await topicService.getTopics(1);
-
-      expect(hubMenuRepo.tagsMenu).toHaveBeenCalledWith(1);
-      expect(cmsApi.primaryMenu).toHaveBeenCalledWith(1);
+      expect(cmsApi.getTopics).toHaveBeenCalledWith('berwyn');
```

Comment:
Finicky strikes again. `const ESTABLISHMENT = 'berwyn'` maybe? It crops up a few times 😬

Reply:
meh.. little bit resistant to deduplication through use of constants in tests. Tend to like to see realistic examples in test data and use of constants can obscure that.

Not the strongest argument granted :-)